sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/AggregationIterator.scala

```diff
@@ -110,16 +118,16 @@ abstract class AggregationIterator(
     functions
   }
 
-  // Positions of those non-algebraic aggregate functions in allAggregateFunctions.
```

Comment:
Throughout this file, "non-algebraic" has been replaced with "interpreted". This renaming lets us remove one level of negation in a bunch of places, simplifying a bunch of code.